scripts/calculateKpi/protocols/blaze-the-base/index.ts

```diff
@@ -0,0 +1,31 @@
+import { RedisClientType } from '@redis/client'
+import { KpiResult, NetworkId } from '../../../types'
+import { getBlockRange } from '../utils/events'
+import { fetchNetworkMetrics } from '../utils/networks'
+
+export async function calculateKpi({
```

Comment:
nit: we should document this function like we did for the other campaigns (KPI unit, business purpose, calculation method, etc).

This affects the public docs we publish about this: https://docs.divvi.xyz/protocol/calculate-kpi/